src/layup/orbitfit.py

```diff
@@ -65,6 +65,11 @@ def _orbitfit(data, cache_dir: str):
     if len(data) == 0:
         return np.array([], dtype=_RESULT_DTYPES)
 
+    # sort the observations by the obstime if specified by the user
+
+    if sort_array:
+        data = np.sort(data, order="obstime")
```

Comment:
We could modify the sorting algorithm to be `heapsort` which _should_ have a faster run time, and since stable sort isn't required, it might be better. Though, we may discover that the average size of the data set is small enough that it doesn't really matter. 

```suggestion
        data = np.sort(data, order="obstime", kind="mergesort")
```